assets/src/components/reorderer/edit.css

```diff
@@ -34,6 +34,7 @@
 .amp-story-reorderer .components-disabled div:first-of-type {
 	padding: 0;
 	width: 100%;
+	height: 100%;
```

Comment:
The ratio of the page preview when reordering seems to differ from the editor (the inner width/height ratio, not considering the borders). It's around 0.52 for reordering and around 0.59 for the editor. Do you think it would make sense to change that, too?

Reply:
Totally.

Looks like I've mentioned that before in https://github.com/ampproject/amp-wp/issues/1967#issuecomment-473848183, but seems that it got lost under the pile of other comments and issues.

Reply:
Updated it to 166x277. 166 / 277 = 0.5992779783

Reply:
The 166/277 seems to be the outer width and height ratio, however, in case of the editor 328/553 is the inner width and height ratio -- considering just the room available for the blocks and not the border surrounding the page. Probably the inner ratio should match instead?

Also, the `--preview-scale: 0.42;` might need adjusting with the new values.

Reply:
Looks like scale might be 0.47 now (155/338).

Reply:
Yup, looks like it!